### PR TITLE
feat(context): added 'context delete' command

### PR DESCRIPTION
**Pull request by YuryShkoda**

### CODE REVIEW

File: src/sasjs-context/index.js [hunk 1]
```diff
@@ -235,6 +249,45 @@ async function edit(config, target) {
...
+        console.log(
+          chalk.redBright('An error has occurred when processing context.', err)
+        )
+      }
+    })
+
+  if (deletedContext) {
+    console.log(
+      chalk.greenBright.bold.italic(
+        `Context with an id '${contextId}' has been deleted!`
```
Comment (allanbowe) [inline src/sasjs-context/index.js:285]: should be "the id" or just "with id"

**Review by allanbowe — APPROVED** [review]

File: src/sasjs-context/index.js (hunk 1 above)
Comment (YuryShkoda) [reply src/sasjs-context/index.js]: fixed

File: src/sasjs-context/index.js [hunk 2]
```diff
@@ -235,6 +249,45 @@ async function edit(config, target) {
...
+    .deleteContext(contextId, accessToken)
+    .catch((err) => {
+      if (err.hasOwnProperty('body')) {
+        const body = JSON.parse(err.body)
+        const message = body.message || ''
+        const details = body.details || ''
+
+        console.log(
+          chalk.redBright(
+            'An error has occurred when processing context.',
```
Comment (krishna-acondy) [inline src/sasjs-context/index.js:271]: Might be more useful to say `An error has occurred when deleting context ${id}.`, or even just `An error has occurred when deleting context.`

Comment (YuryShkoda) [reply src/sasjs-context/index.js]: agree, fixed

**Review by krishna-acondy — APPROVED** [review]

**Comment by YuryShkoda** [comment]
closes #74

**Review by krishna-acondy — APPROVED** [review]

**Pull request merged**